src/main/java/ch/njol/skript/bukkitutil/SoundUtils.java

```diff
@@ -25,19 +26,17 @@ public final class SoundUtils {
 		soundString = soundString.toUpperCase(Locale.ENGLISH);
 		if (SOUND_IS_INTERFACE) {
 			try {
-				//noinspection deprecation
+				//noinspection deprecation,removal
 				return Sound.valueOf(soundString).getKey();
-			} catch (IllegalArgumentException ignore) {
-			}
+			} catch (Exception ignored) {}
 		} else {
 			try {
 				//noinspection unchecked,rawtypes
 				Enum soundEnum = Enum.valueOf((Class) Sound.class, soundString);
 				return ((Keyed) soundEnum).getKey();
-			} catch (IllegalArgumentException ignore) {
-			}
+			} catch (IllegalArgumentException ignored) {}
 		}
-		return null;
+		return NamespacedUtils.getNamespacedKey(soundString.toLowerCase(Locale.ENGLISH), false);
```

Comment:
I would rather you simplify the parsing done in EffPlaySound to use this method. This is not doing the same level of validation as before (with the key pattern and such)

Reply:
I'm not sure I know what you mean.

Reply:
Nevermind, I misunderstood this 🤦